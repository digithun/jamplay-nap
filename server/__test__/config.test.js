/* eslint-env jest */

process.env.MAILGUN_API_KEY = 'FOO_MAILGUN_API_KEY'
process.env.MAILGUN_DOMAIN = 'BAR_MAILGUN_DOMAIN'

process.env.BASE_URL = 'http://127.0.0.1:8080'

process.env.AUTH_LOCAL_URI = 'http://localhost:4000/auth/local'
process.env.AUTH_RESET_URI = 'http://localhost:4000/auth/reset'
process.env.AUTH_NEW_RESET_URI = 'http://localhost:4000/auth/reset'
process.env.AUTH_VERIFIED_URI = 'http://localhost:4000/auth/verified'

process.env.STATIC_RESOLVE_URL = 'foo'
process.env.SHARE_IMAGE_SERVICE_URL = 'bar'
process.env.SHARE_IMAGE_SERVICE_API_KEY = 'baz'
process.env.OPTICS_API_KEY = 'bas'

process.env.SESSIONS_TTL = 0

const config = require('../config')

describe('config', () => {
  it('should return config', () => {
    expect(config).toMatchSnapshot()
  })
})
