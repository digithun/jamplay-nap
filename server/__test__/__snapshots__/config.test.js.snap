// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config should return config 1`] = `
Object {
  "achievement_service_api_key": undefined,
  "achievement_service_url": undefined,
  "auth_change_email_uri": "/auth/change-email",
  "auth_error_uri": "/auth/error",
  "auth_local_uri": "http://localhost:4000/auth/local",
  "auth_new_reset_uri": "http://localhost:4000/auth/reset",
  "auth_reset_uri": "http://localhost:4000/auth/reset",
  "auth_validate_reset_uri": "/auth/validate-reset",
  "auth_verified_uri": "http://localhost:4000/auth/verified",
  "base_url": "http://127.0.0.1:8080",
  "bigquery_service_endpoint": null,
  "cookie_secret": "foo",
  "dev": true,
  "e_wallet_api": undefined,
  "e_wallet_enabled": false,
  "graphiql_enabled": true,
  "graphql_disabled": false,
  "is_optics_enabled": false,
  "jwt_secret": "foo",
  "mailgun_api_key": "FOO_MAILGUN_API_KEY",
  "mailgun_domain": "BAR_MAILGUN_DOMAIN",
  "mongo_url": "mongodb://mongo/graphql",
  "mubsub": undefined,
  "mubsub_enabled": false,
  "next_disabled": false,
  "passport_disabled": false,
  "redis_url": "redis://redis",
  "sessions_ttl": 604800000,
  "share_image_service_api_key": "baz",
  "share_image_service_url": "bar",
  "static_resolve_url": "foo",
  "tracing_enabled": true,
  "twitter_consumer_key": undefined,
  "twitter_consumer_secret": undefined,
}
`;
